src/TemplateLayoutPathResolver.js

```diff
@@ -82,9 +79,7 @@ class TemplateLayoutPathResolver {
   getFullPath() {
     if (!this.filename) {
       throw new Error(
-        `You’re trying to use a layout that does not exist: ${
-          this.originalPath
-        } (${this.filename})`
+        `You’re trying to use a layout that does not exist: ${this.originalPath} (${this.filename})`
```

Comment:
This is a prettier format

Reply:
Hello @zachleat 

I am using the local installed prettier 1.18.2 and running it will not change this line.
I also checked against the online [playground ](https://prettier.io/playground/) - no changes.

When I use line `85-87` then prettier will change it to `82`

Do I miss a prettier configuration somewhere?
Should we add a `.prettierrc` config file? 

What can I do to reproduce your preferred style?